localization/ekf_localizer/CMakeLists.txt

```diff
@@ -46,7 +47,8 @@ if(BUILD_TESTING)
     test/test_mahalanobis.cpp
     test/test_measurement.cpp
     test/test_numeric.cpp
-    test/test_state_transition.cpp)
+    test/test_state_transition.cpp
+    test/test_warning_message.cpp)
```

Comment:
I personally think this test for warning messages is not necessary. What do you think?
(same as https://github.com/autowarefoundation/autoware.universe/pull/2657#discussion_r1080865663)

Reply:
I think we need the test. The message function is rarely executed when running the logging simulator, so it is hard to detect a bug if we wrongly set the string format.

Reply:
The basic concept is "do not believe in human eyes." 
Humans can often make mistakes in very easy and obvious parts. Even if it looks unnecessary, we should add tests.

Reply:
This is too dependent on the implementation, and thus would increase a cost of modifying them in the future (e.g. changing the error messages). Also, I see little advantage of adding this test, since it is essentially a test for fmt::format.

But this is just my opinion, so I will leave the decision up to @YamatoAndo @TakaHoribe .

Reply:
As kminoda-san said, I don't think it is necessary to make a function to test if the message content is as intended.
But since IshitaTakeshi-san has already made it, I think we can merge this PR.